api-report/fluid-static.api.md

```diff
@@ -6,11 +6,11 @@
 
 import { AttachState } from '@fluidframework/container-definitions';
 import { BaseContainerRuntimeFactory } from '@fluidframework/aqueduct';
-import { Container } from '@fluidframework/container-loader';
 import { DataObject } from '@fluidframework/aqueduct';
 import { IAudience } from '@fluidframework/container-definitions';
 import { IChannelFactory } from '@fluidframework/datastore-definitions';
 import { IClient } from '@fluidframework/protocol-definitions';
+import { IContainer } from '@fluidframework/container-definitions';
```

Comment:
there's already an import for this package in L10 above

Reply:
I believe the API report automatically separates each import into its own line. They're on the same line in the code itself

Reply:
actually wait this is the .md file how does this work 😵

Reply:
These files are auto-generated. See <https://github.com/microsoft/FluidFramework/wiki/API-reports-and-review>.